requirements-test-full.txt

```
@@ -1,7 +1,7 @@
 fsspec>=2022.11.0;sys_platform != "win32"
 jax[cpu]>=0.2.15;sys_platform != "win32" and python_version < "3.13"
 numba>=0.50.0;sys_platform != "win32" and python_version < "3.13"
-numexpr>=2.7; python_version < "3.13"
+numexpr>=2.7
```

```suggestion
numexpr>=2.7; python_version < "3.14"
```

Would this make sense so we can test on 3.14 soon?

---

I don't know. Whatevere you suggest here